programs/drift/src/state/events.rs

```diff
@@ -256,10 +256,15 @@ pub struct OrderActionRecord {
     pub maker_existing_base_asset_amount: Option<u64>,
     /// precision: PRICE_PRECISION
     pub trigger_price: Option<u64>,
+
+    /// the idx of the builder in the taker's [`RevenueShareEscrow`] account
+    pub builder_idx: Option<u8>,
+    /// precision: QUOTE_PRECISION builder fee paid by the taker
+    pub builder_fee: u64,
```

Comment:
why not have this be an option? also make sure you're careful about not adding this to event IDL until after deploy goes out